braidpool_spec.md

```diff
@@ -12,6 +12,10 @@ encrypted miner communication is being pursued by the
 [StratumV2](https://github.com/stratum-mining/sv2-spec) project, which Braidpool
 will build upon.
 
+[kp] We don't need stratum v2 at all. If the grinding machines talk to a
+braidpool controller hosted on their facility, then the need for a secure
+communication layer to a pool operator disappears. [/kp]
+
```

Comment:
Correct. But let's imagine a more dynamic scenario with e.g. multiple sites for one entity who runs only one braidpool instance instead of one per facility.

There are lots of considerations in StratumV2 that I want to punt on wherever possible, and establish synergy with them instead of competition.